tests/test_routes.py

```diff
@@ -140,3 +140,10 @@ def test_read_order(self):
         self.assertEqual(resp.status_code, status.HTTP_200_OK)
         data = resp.get_json()
         self.assertEqual(data["customer_name"], order.customer_name)
+
+    def test_delete_order(self):
+        """It should Delete an entire order by order id"""
+        order = self._create_orders(1)[0]
+        resp = self.client.delete(f"{BASE_URL}/{order.id}")
+        self.assertEqual(resp.status_code, status.HTTP_204_NO_CONTENT)
+        print("Success")
```

Comment:
No need to print "Success" here

Reply:
Sorry, my bad, that was for debugging purposes